src/App/Pages/Settings/SettingsPage/SettingsPageViewModel.cs

```diff
@@ -374,6 +374,10 @@ public async Task ApproveLoginRequestsAsync()
 
         public async Task VaultTimeoutActionAsync()
         {
+            if (!string.IsNullOrEmpty(_policyService.GetPolicyString(_vaultTimeoutPolicy, "action")))
+            {
+                return;
+            }
```

Comment:
I know we're already using hardcoded strings to get policies, but could we have a place where all the Policies are as constants. It could be in the `Policy` class.
This one in particular is being used more than once so IMO it makes sense to extract it to a constant somewhere.
In order not to pollute this PR, we could add a tech debt task to move all the others hardcoded policy strings into constants as well.
Also, I wanted to add that if we have all the strings as constants in one file it's easier to see which policies are being implemented in this repo and which are not.

Reply:
Yes, IMO they should be in the `Policy` class because I think that class is responsible of knowing which are the valid string values of policies. I think a tech debt task for this would be good so we put all the policies in one place 😄